chipsec/cfg/8086/cml.xml

```diff
@@ -8,10 +8,6 @@
   <!--                                      -->
   <!-- #################################### -->
   <info family="core" detection_value="A0652, A0653, A0655, A0660, A0661">
-    <sku did="0x3E20" name="CometLake" code="CML" longname="CometLake H8 Core / XeonW" />

```

Comment:
What's the reason for removing these DIDs?

Reply:
Comet Lake and Whiskey Lake CPUID and DID collision, refer CHIPSEC-803